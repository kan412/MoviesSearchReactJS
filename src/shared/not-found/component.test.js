import React from 'react';
import { shallow } from 'enzyme';
import NotFound from './component';

describe('<NotFound />', () => {
  it('renders without breaking', () => {
    const component = shallow(<NotFound />);
    expect(component).toMatchSnapshot();
  });
});
