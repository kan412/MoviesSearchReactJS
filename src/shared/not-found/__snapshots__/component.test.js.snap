// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NotFound /> renders without breaking 1`] = `
<div
  className="error"
>
  <h1
    className="title"
  >
    404! Page not found
  </h1>
  <p
    className="info"
  >
    <span>
      Please go back to 
    </span>
    <a
      href="/"
    >
      main page
    </a>
    <span>
       and try again. Thank you
    </span>
  </p>
</div>
`;
